MockData.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
```

Comment:
What is the use of this class? Remove this class if not used.

Reply:
We have removed the class. It was created by mistake.